ImportPackDialog.gd

```diff
@@ -0,0 +1,20 @@
+extends FileDialog
+const FileHandler = preload("res://FileHandler.gd")
```

Comment:
FileHandler should probably be a static class which extends nothing if it is just being used like a library. This means it does not need to be initialized with `.new()` either.